charts/rancher-eks-operator/1.0.801/questions.yaml

```diff
@@ -0,0 +1 @@
+rancher_min_version: 2.5.6-alpha1
```

Comment:
This should be 2.5.10 and last chart should max at 2.5.9

Reply:
This is done.